.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Description
+
+Closes: *#XXXX*
+
+*Add a description of the changes that this PR introduces and the files that
+are the most critical to review.*
+
+---
+
+### Author Checklist
+
+*All items are required. Please add a note to the item if the item is not applicable and
+please add links to any relevant follow up issues.*
+
+I have...
+
+* [ ] Targeted the correct branch
+* [ ] Included the necessary unit and e2e tests
```

Comment:
let's make this one
```suggestion
* [ ] Included the necessary unit tests
```
since we don't have any e2e ones (or rather, those are the interchain tests)